### ISSUE TITLE
Cannot redirect to a URL of different domain while SPA is enabled 

### ISSUE THREAD

**Issue by vbence86** [liferay/senna.js#223]
### Create test bundles 
Set up two server instances  to be run at the same time
In the **first instance (8080)**, create a regular page named page1
In the **second instance (8081)**, create a Link to URL page named page1 that links to http://localhost:8080/examples/amd/page1.html

##### Nginx (on Mac)
```bash
brew install nginx
```
##### Configure the proxy settings in ```/usr/local/etc/nginx/nginx.conf```
```bash
...
        location / {
            root   /absolute/path/to/senna.js;
            index  index.html index.htm;
        }
        rewrite /redirect http://localhost:8080/examples/amd/page1.html redirect;
...
```
### Configure Senna JS
I've used the AMD API, but this is not relevant in terms of the reported issue
```html
<a href="/redirect">Redirect to http://localhost:8080/examples/amd/page1.html</a>
```
```javascript
var app = new senna.App();
      app.addSurfaces('mySurface');
      app.addRoutes([
        {
          path: '/redirect',
          handler: senna.HtmlScreen
        }
      ]);
```
### Excetute the first test server instance
Simply enter to Senna JS project folder and run ```gulp server```. This will kick off a server that listens on localhost:8080. 

### Manual test

- Make sure all servers respond individually as expected 
- Open localhost:8081/examples/amd/yourfile.html
- Click on the link that attempts to open ```/redirect```
- Check the network activity with dev tools of your choice. 

### Result
Assert Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:8080/web/guest/page1. (Reason: CORS header 'Access-Control-Allow-Origin' missing). error received in browser console or something similar.

**Comment by brunobasto** [comment]
Hey @vbence86,

Can you confirm that this was fixed by: https://github.com/liferay/senna.js/releases/tag/v2.4.2?

This should also fix https://github.com/liferay/senna.js/issues/120, right?

**Comment by vbence86** [comment]
Hi @brunobasto,

Yes, I can confirm that this fix has been tested and fixes the above along with #120.

Thanks,
Bence

**Comment by brunobasto** [comment]
Awesome! Thanks
